supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -512,13 +513,13 @@ There is no functional difference between the first server that was installed an
 [discrete]
 [[intel-64]]
 ==== image:images/yes.png[yes] Intel 64 (noun)
-*Description*: The Intel 64-bit version of the x86 architecture. Use this format when referring to information that is exclusive to Intel processors. Only use in RHEL documentation. 
+*Description*: The Intel 64-bit version of the x86 architecture. Use this format when referring to information that is exclusive to Intel processors. Only use in RHEL documentation.
```

Comment:
I left another 'only' comment below but here I think it should be:
`Use only in RHEL documentation.`

Reply:
Agree